aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -0,0 +1,174 @@
+.alfresco-forms-controls-MultiSelect {
+   border: 1px solid #666;
```

Comment:
Although they're not fully formed, we should be looking at re-using the LESS variables that are known and looking to abstract new ones where applicable. There is border variables, it may or may not be appropriate to use here, but there are no LESS variables referenced at all here.